ocaml/idl/datamodel.ml

```diff
@@ -6223,7 +6223,7 @@ let vdi =
          field ~in_oss_since:None ~in_product_since:rel_miami ~ty:String ~qualifier:DynamicRO ~default_value:(Some (VString "")) "location" "location information";
          field ~in_oss_since:None ~ty:Bool ~qualifier:DynamicRO "managed" "";
          field ~in_oss_since:None ~ty:Bool ~qualifier:DynamicRO "missing" "true if SR scan operation reported this VDI as not present on disk";
-         field ~in_oss_since:None ~ty:(Ref _vdi) ~qualifier:DynamicRO "parent" "References the parent disk, if this VDI is part of a chain";
+         field ~in_oss_since:None ~ty:(Ref _vdi) ~qualifier:DynamicRO ~lifecycle:[Published, rel_rio, ""; Deprecated, rel_ely, "The field was never used."] "parent" "References the parent disk, if this VDI is part of a chain. This field is deprecated";
```

Comment:
Given that this doesn't hold a reference to the parent, I think the description should not say it does.
Instead we should change the description to something like "Always null. Deprecated."